### PR TITLE
add ETMP chain

### PR DESCRIPTION
**Pull request by LuxesOwen**
Add ETMP chain,use chainId=48 for Mainnet,chainId=49 for Testnet

### CODE REVIEW

**Comment by MoGenius5420** [comment]
`gh pr checkout 1654`

**Comment by MoGenius5420** [comment]
`gh pr checkout 1654`

**Review by MoGenius5420 — APPROVED** [review]

**Review by MoGenius5420 — APPROVED** [review]

**Comment by MoGenius5420** [comment]
@mogeeeenius - Instagram

**Comment by LuxesOwen** [comment]
@ligi Please merge, thank you very much

File: _data/chains/eip155-49.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "infoURL": "https://etm.network",
+  "shortName": "etmp",
+  "chainId": 49,
+  "networkId": 49,
+  "icon": "etmp",
+  "explorers": [
+    {
+      "name": "etmp",
+      "url": "https://pioneer.etmscan.network/",
```
Comment (ligi) [inline _data/chains/eip155-49.json:22]:
```suggestion
      "url": "https://pioneer.etmscan.network",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by LuxesOwen** [comment]
hi @ligi , thanks for your check,please merge my request.

**Pull request closed without merging**